src/main/helper/ApiFormatter.ts

```diff
@@ -162,7 +164,7 @@ export function fromApiFormat(fromApiCaseData: CaseApiDataResponse, req?: AppReq
     ),
     acknowledgementOfClaimLetterDetail: setDocumentValues(
       fromApiCaseData?.case_data?.servingDocumentCollection,
-      acceptanceDocTypes
+      setAcceptanceDocTypes(fromApiCaseData?.case_type_id)
```

Comment:
might be better to name it getAcceptanceDocTypes here as we are not really setting anything, but just thinking why not just do fromApiCaseData?.case_type_id === CaseTypeId.ENGLAND_WALES ? acceptanceDocTypesEW : acceptanceDocTypes;